assets/js/services/team_builder.js

```diff
@@ -0,0 +1,34 @@
+app.factory("TeamBuilder", ["Team", function(Team) {
+
+
+  var createTeam = function(name, cb) {
+    var team = new Team({name: name});
+    team.$save(cb);
+  };
+
+  var team = function(id) {
+    return Team.get({id: id});
+  };
+
+  var addUser = function(teamId, user, cb) {
+    var team = Team.get({id: teamId}, function(data) {
+      data.users.push(user);
+      data.$update(cb);
+    });
+  };
+
+  var addAdmin = function(teamId, user, cb) {
```

Comment:
If the team has already been loaded beforehand, can we have a function such as:
`
var addAdmin = function(team, user, cb) {
team.admins.push(user);
team.users.push(user);
team.$update(cb);
};
`

Or should we actually have a backend route for this:

`POST '/team/5/add-admin'`
we can send the admin id as data in the post request